@kiva/kv-components/vue/stories/KvTextInput.stories.js

```diff
@@ -157,11 +158,18 @@ export const WidthSet = (args, {
 export const Disabled = DefaultTemplate.bind({});
 Disabled.args = {
 	disabled: true,
+	canClear: false,
```

Comment:
I think this can be removed since the default of canClear is false

Reply:
Done